Tools/CISupport/ews-build/config.json

```diff
@@ -227,28 +227,56 @@
       "factory": "macOSBuildFactory", "platform": "mac-monterey",
       "configuration": "release", "architectures": ["x86_64", "arm64"],
       "triggers": ["api-tests-mac-ews", "macos-monterey-release-wk1-tests-ews", "macos-monterey-release-wk2-tests-ews", "macos-release-wk2-stress-tests-ews"],
-      "workernames": ["ews101", "ews102", "ews103", "ews105","ews118", "ews119", "ews120", "ews141", "ews161", "ews162"]
+      "workernames": ["ews101", "ews102", "ews103", "ews105", "ews119", "ews161", "ews162"]
+    },
+    {
+      "name": "macOS-Ventura-Release-Build-EWS", "shortname": "mac", "icon": "buildOnly",
+      "factory": "macOSBuildFactory", "platform": "mac-monterey",
```

Comment:
Should this platform be `mac-ventura`? Same for `macOS-AppleSilicon-Ventura-Release-WK1-Tests-EWS`

Reply:
Yes it should. Thank you! Fixed.